merlin-server/Dockerfile

```diff
@@ -1,7 +1,12 @@
-FROM eclipse-temurin:17-focal
+FROM alpine:3.15
```

Comment:
From the PR description, I was expecting to see `eclipse-temurin:17-jre-focal` here. Am I misunderstanding something?

Reply:
Joel is using [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/).

Reply:
Whoa cool, thanks, I missed the second "FROM" line below

Reply:
Yup, that's correct. I'm using alpine as a tiny linux container to do the tar extraction in, and `eclipse-temurin:17-jre-focal` is used for the second phase.